src/autotype/AutoTypeSelectDialog.cpp

```diff
@@ -71,7 +71,7 @@ AutoTypeSelectDialog::AutoTypeSelectDialog(QWidget* parent)
 
     m_ui->searchCheckBox->setShortcut(Qt::CTRL + Qt::Key_F);
     connect(m_ui->searchCheckBox, &QCheckBox::toggled, this, [this](bool checked) {
-        Q_UNUSED(checked);
+        m_searchTimer.setInterval(checked ? 150 : 0);
```

Comment:
This logic needs to be duplicated in `AutoTypeSelectDialog::setMatches` because it prevents the signal from being fired on initial setup when the dialog is opened to prevent double search.

Reply:
Doesn't the signal get blocked in setMatches and an explicit call to perform search is made?

Reply:
Yes, but doesn't that also mean that it call setInterval when the check is grayed out but checked?

Reply:
Good point